repository/src/main/java/com/epam/esm/entity/Order.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.hibernate.envers.Audited;
```

Comment:
Its forbidden to use any Hibernate specific features.

Reply:
All Hibernate specific features was deleted from the code